src/io.rs

```diff
@@ -50,3 +70,27 @@ pub fn read_embeddings_view(
 
     Ok(embeds?)
 }
+
+pub fn read_quantized_embeddings(
+    filename: &str,
+    embedding_format: QuantizedEmbeddingFormat,
+) -> Result<Embeddings<VocabWrap, StorageWrap>, Error> {
+    let f = File::open(filename).context("Cannot open embeddings file")?;
+    let mut reader = BufReader::new(f);
+
+    use self::QuantizedEmbeddingFormat::*;
+    let embeds: Embeddings<VocabWrap, StorageWrap> = match embedding_format {
+        FinalFusion => {
+            let quantized_embeds: Embeddings<VocabWrap, QuantizedArray> =
+                ReadEmbeddings::read_embeddings(&mut reader)?;
+            quantized_embeds.into()
```

Comment:
Since we don't have ```from()``` functions converting ```Embeddings<VocabWrap, _>``` into ```Embeddings<VocabWrap, StorageWrap>```, I'm thinking maybe we can add them to ```finalfuaion-rust/embeddings.rs```?

Reply:
But if so there would be a even longer list of functions for all possible combinations......

Reply:
Or I can also ```into_parts``` and then construct a new ```Embeddins```

Reply:
I think I'll just make a PR on ```finalfusion-rust`` converting ```Embeddings<VocabWrap, _>``` into ```Embeddings<VocabWrap, StorageWrap>```... seems more continuous.